src/test/java/org/folio/rest/support/builders/HoldingRequestBuilder.java

```diff
@@ -38,7 +40,8 @@ private HoldingRequestBuilder(
     String callNumber,
     String callNumberPrefix,
     String callNumberSuffix,
-    String hrid) {
+    String hrid,
+    String callNumberTypeId) {
```

Comment:
This is just aesthetic, but I think we should keep the `callNumber*` components together in the argument list, i.e. swap `hrid` and `callNumberTypeId`.